corehq/apps/reports/util.py

```diff
@@ -512,23 +512,29 @@ def get_matching_tableau_users_from_other_domains(user):
 
 
 @atomic
-def add_tableau_user(domain, username):
+def add_tableau_user(domain, username, blocking_exception=False):
```

Comment:
I noticed `blocking_exception=True` wasn't passed as arguments to where `add_tableau_user` is called (such as in `add_domain_membership` and `undo`). Couldn't it also be an issue there?

Also, just a thought - would it make sense to handle the exception on the calling function as opposed to introducing this parameter?

Reply:
Though if the param is never set to `True`, then you could probably just remove it, right?